docs/guides/validation-user-guide.md

```diff
@@ -4,4 +4,461 @@ headline: Validation User Guide
 bodyclass: docs
 layout: docs
 ---
-<p class="coming-soon">Coming soon...</p>
+
+Protobuf is a great tool for structuring domain models. Protobuf definitions help us to standardize
+creating [Value Objects](https://martinfowler.com/bliki/ValueObject.html).
```

Comment:
Not only value objects. We also define entity states. We also define signals that are immutable, but not value objects per se.

Reply:
I would start with repeating the fact that in Spine projects data model is defined in Protobuf, and which parts of the model are defined using this language. Then say that we need to make sure that a model is consistent with business rules, etc.